manifests/pip/bootstrap.pp

```diff
@@ -19,12 +21,19 @@
       'AIX' => '/opt/freeware/bin',
       default => '/usr/bin'
     }
+
+    $environ = $facts['os']['family'] ? {
+      'AIX' => [ "http_proxy=${http_proxy}", "https_proxy=${http_proxy}" ],
+      default => [ "HTTP_PROXY=${http_proxy}", "HTTPS_PROXY=${http_proxy}" ]
```

Comment:
I wonder if some users have already been setting these environment variables outside of this module.  This would unset them.  Maybe best to only set `environment` if an http_proxy has been specified.
```puppet
if $http_proxy { # Assuming we change the default to `undef`
  $environ = $facts['os']['family'] ? {
    'AIX'   => [ "http_proxy=${http_proxy}", "https_proxy=${http_proxy}" ],
    default => [ "HTTP_PROXY=${http_proxy}", "HTTPS_PROXY=${http_proxy}" ],
  }
} else {
  $environ = undef
}
```

Reply:
Done.